### PR TITLE
Whitelabel fixes

### PR DESCRIPTION
**Pull request by jarekzet**
Whitelabel fixes according to https://docs.cyfronet.pl/pages/viewpage.action?spaceKey=FID&title=GUI+System+dla+EOSC+Marketplace

### CODE REVIEW

**Review by martaswiatkowska — CHANGES_REQUESTED** [review]
.idea folder should be added to gitingnore

File: .idea/egi-marketplace.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (goreck888) [inline .idea/egi-marketplace.iml]: This file is RubyMine project file. It needs to be ignored in .gitignore file. I suggest to add whole /.idea/ folder.

File: .idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (goreck888) [inline .idea/misc.xml]: Similar as above

File: views/home/_products.html.haml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  #tab-2.tab-content-home{ "data-target" => "home-page.tab" }
+    .container
+      .row
+        = render "home/products/cards",
+            cards: research_areas,
+            url: -> (ra) { services_path(research_areas: ra.to_param) },
+            default_logo: "ico_ra_default.png"
+
+.row.d-flex.justify-content-center.mt-4
+  = link_to "Browse all services", services_path, class: "button button--primary"
```
Comment (goreck888) [inline views/home/_products.html.haml:20]: I suppose the class is `button-primary` - with one dash but maybe I'm wrong

File: views/layouts/_footer.html.haml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    %hr.mb-4
+    .row
+      .col-md
+        %ul
+          %li
+            .title About
+          %li.mb-1
+            %a{ href: "#{portal_base}/about/eosc" } EGI
+          %li.mb-1
+            %a{ href: "#{portal_base}/about/eosc-doers" } EGI Doers
```
Comment (goreck888) [inline views/layouts/_footer.html.haml:22]: I suppose these links should be fixed by e.g. environmental variable. For the first public show i think it's ok but we need to refactor that

File: views/layouts/_head.html.haml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  %meta{ content: "width=device-width, initial-scale=1, shrink-to-fit=no", name: "viewport" }/
+  %title EGI Marketplace
+  = csrf_meta_tags
+  = csp_meta_tag
+  = favicon_pack_tag "favicon.ico"
+  = favicon_pack_tag "favicon-180x180.png", rel: "apple-touch-icon", type: "image/png"
+  = stylesheet_pack_tag "application"
+  = javascript_pack_tag "application"
+  - if Rails.env.production?
+    = render "layouts/matomo"
```
Comment (goreck888) [inline views/layouts/_head.html.haml:13]: I'm not sure if we should provide matomo tracking and generally our trackers to these instances. @bwilk, @agpul ?

**Review by goreck888 — CHANGES_REQUESTED** [review]

**Comment by goreck888** [comment]
Add .gitignore file, you can copy it from source mp repo.

File: views/home/_products.html.haml (hunk 3 above)
Comment (jarekzet) [reply views/home/_products.html.haml:20]: no, double dash is OK

**Comment by jarekzet** [comment]
Added .gitignore file

**Comment by martaswiatkowska** [comment]
delete .idea folder from this pr but not your local

**Comment by goreck888** [comment]
Do it by `git rm -r .idea` check in `git status` if files are removed and then commit changes.

**Comment by jarekzet** [comment]
.idea folder removed

File: views/layouts/_head.html.haml (hunk 5 above)
Comment (goreck888) [reply views/layouts/_head.html.haml:13]: You can comment or remove everything from line 12 to line 15 and I'm not sure if there is a lack of free line at the end

**Review by goreck888 — APPROVED** [review]
Just disable trackers in head file

**Pull request merged**